rstr/rstr_base.py

```diff
@@ -114,7 +114,7 @@ def __getattr__(self, attr):
 
     def sample_wr(self, population, k):
         '''Samples k random elements (with replacement) from a population'''
-        return [self._random.choice(population) for i in itertools.repeat(None, k)]
+        return [self._random.choice(population) for _ in range(k)]
```

Comment:
`itertools.repeat()` is slightly more efficient for this use-case, because we don't care about the integers that `range()` would generate. This is [one of the recommended uses for `itertools.repeat()`.](https://stackoverflow.com/a/9098860/845210)

```python
import timeit
repeat=100_000
result1 = timeit.timeit('[random.choice(string.printable) for _ in itertools.repeat(None, 100)]', setup='import random, string, itertools', number=repeat)
print(f'Itertools took {result1:.5}')
# Itertools took 5.0574
result2 = timeit.timeit('[random.choice(string.printable) for _ in range(100)]', setup='import random, string', number=repeat)
print(f'Range took {result2:.5}')
# Range took 5.2794
```